src/OpenApi/ClientGenerator.php

```diff
@@ -258,27 +262,32 @@ protected function extractArguments(array $parameters): array
 
                     if (isset($propDefinition['type']) && $propDefinition['type'] == 'array') {
                         $itemsType = $this->getType($propDefinition['items']['$ref']);
-                        $argument['doc'] = 'Model\\' . $itemsType . '[]';
+                        $argument['doc'] = 'Model\\'.$itemsType.'[]';
```

Comment:
On several lines the spaces before and after a `.` are removed. Please change this back.

Reply:
potato patata. guess my IDE did this. was not aware of it. will revert it.

Reply:
potato patata. guess my IDE did this. was not aware of it. will revert it.